.github/workflows/ci.yml

```diff
@@ -45,15 +45,15 @@ jobs:
       - name: PHPCompatibility
         uses: pantheon-systems/phpcompatibility-action@v1
         with:
-          test-versions: 7.4-
+          test-versions: 8.2
   build_test:
     strategy:
       matrix:
-        drupal-version: [ 10 ]
+        drupal-version: [ 10, 11 ]
```

Comment:
If we claim D9 compatibility, I think we should also test there. I see some type changes that make me nervous about D9 :(